cypress/support/pages/yves/cart/cart-page.ts

```diff
@@ -56,6 +69,36 @@ export class CartPage extends YvesPage {
       form.submit();
     }
   };
+
+  addFirstCartItemNote = (params: CartItemNoteAddParams): void => {
+    this.repository.getFirstCartItemNoteField().type(params.message);
+  };
+
+  clearFirstCartItemNote = (): void => {
+    this.repository.getFirstCartItemNoteField().clear();
+  };
+
+  submitFirstCartItemNote = (): void => {
+    this.repository.getFirstCartItemNoteSubmitButton().click();
+
+    this.repository.getPageLayoutCartAjaxLoader().should('be.not.visible');
+  };
+
+  getFirstCartItemNoteField = (): Cypress.Chainable => {
+    return this.repository.getFirstCartItemNoteField();
+  };
+
+  getCartSummary(): Cypress.Chainable {
```

Comment:
try to be consistent with methods above - `getCartSummary = (): Cypress.Chainable => {}`